Formula/e/evil-helix.rb

```diff
@@ -0,0 +1,29 @@
+class EvilHelix < Formula
+  desc "Soft fork of the helix editor"
+  homepage "https://github.com/usagi-flow/evil-helix"
+  url "https://github.com/usagi-flow/evil-helix/archive/refs/tags/release-20250104.tar.gz"
+  sha256 "b03c78ea4cacd11eac8c2fb2e40c4a20e5d6d29ab151e3176876bf58c298b7b5"
+  license "MPL-2.0"
+  head "https://github.com/usagi-flow/evil-helix.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hex", because: "both install `hx` binaries"
+  conflicts_with "helix", because: "both install `hx` binaries"
+
+  def install
+    ENV["HELIX_DEFAULT_RUNTIME"] = libexec/"runtime"
+    system "cargo", "install", "-vv", *std_cargo_args(path: "helix-term")
+    rm_r "runtime/grammars/sources/"
+    libexec.install "runtime"
+
+    bash_completion.install "contrib/completion/hx.bash" => "hx"
+    fish_completion.install "contrib/completion/hx.fish"
+    zsh_completion.install "contrib/completion/hx.zsh" => "_hx"
+  end
+
+  test do
+    assert_match "post-modern text editor", shell_output("#{bin}/hx --help")
```

Comment:
can you update to do version check instead?

Reply:
Again, copied from helix formula...

Reply:
Added a test for the version.